example/screens/Chatroom/ChatroomScreen.tsx

```diff
@@ -136,16 +137,16 @@ export function ChatroomScreen() {
     await scrollToBottom();
     console.log('after custom scroll to bottom');
   };
-  const customExploreFeedRenderFooter = async () => {
-    console.log('before custom render footer');
-    await renderFooterExploreFeed();
-    console.log('after custom render footer');
-  };
-  const customHandleLoadMore = async () => {
-    console.log('before handle load more');
-    await handleLoadMore();
-    console.log('after handle load more');
-  };
+  // const customExploreFeedRenderFooter = async () => {
+  //   console.log('before custom render footer');
+  //   await renderFooterExploreFeed();
+  //   console.log('after custom render footer');
+  // };
+  // const customHandleLoadMore = async () => {
+  //   console.log('before handle load more');
+  //   await handleLoadMore();
+  //   console.log('after handle load more');
+  // };
```

Comment:
add these commented custom methods in a doc, so it will help in customizability testing and remove it from here